core-web/package.json

```diff
@@ -200,6 +200,7 @@
         "@types/webpack": "4.41.21",
         "@typescript-eslint/eslint-plugin": "6.21.0",
         "@typescript-eslint/parser": "6.21.0",
+        "autoprefixer": "^10.4.0",
```

Comment:
why?

Reply:
While i was testing the lib, I accidentally remove `node_modules` from `core-web`
Then i ran yarn and this appeared
Idk why, but i removed it for now